editor.html

```diff
@@ -226,6 +226,19 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                         </div>
                     </a>
                 </div>
+                <a href="#" class="roundbutton" id="command-language" tabindex="10" title="Select a language">
```

Comment:
@jaustin @microbit-matt-smith 
We discussed translation button placement a while ago, did we end up with a conclusion?

For this incarnation I think it should probably go before the Help button.
I originally liked the idea to have this inside the Help dropdown, to save button area space, but now that the filesystem has combined Save and Load, it's not such an urgent problem.

Reply:
Yeah, it may make more sense to move the language button inside help or a "more options/settings" button if it comes about in the future since this is a feature that will not be used by everyone.